code/modules/mob/living/simple_animal/slaughter/slaughter.dm

```diff
@@ -0,0 +1,259 @@
+//////////////////The Monster
+
+/mob/living/simple_animal/slaughter
+	name = "Slaughter Demon"
+	real_name = "Slaughter Demon"
+	desc = "You should run."
+	speak_emote = list("gurgles")
+	emote_hear = list("wails","screeches")
+	response_help  = "thinks better of touching"
+	response_disarm = "flails at"
+	response_harm   = "punches"
+	icon = 'icons/mob/mob.dmi'
+	icon_state = "daemon"
+	icon_living = "daemon"
+	speed = 1
+	a_intent = "harm"
+	stop_automated_movement = 1
+	status_flags = CANPUSH
+	attack_sound = 'sound/misc/demon_attack1.ogg'
+	min_oxy = 0
+	max_oxy = INFINITY
+	min_tox = 0
+	max_tox = INFINITY
+	min_co2 = 0
+	max_co2 = INFINITY
+	min_n2  = 0
+	max_n2  = INFINITY
+	minbodytemp = 0
+	maxbodytemp = INFINITY
+	faction = list("slaughter")
+	attacktext = "wildly tears into"
+	maxHealth = 200
+	health = 200
+	environment_smash = 1
+	//universal_understand = 1
+	melee_damage_lower = 30
+	melee_damage_upper = 30
+	see_in_dark = 8
+	see_invisible = SEE_INVISIBLE_MINIMUM
+	var/boost = 0
+	bloodcrawl = BLOODCRAWL_EAT
+
+
+	var/devoured = 0
+
+	var/list/nearby_mortals = list()
+	var/cooldown = 0
+	var/gorecooldown = 0
+	var/vialspawned = FALSE
+	var/playstyle_string = "<B>You are the Slaughter Demon, a terible creature from another existence. You have a single desire: To kill.  \
+						You may Ctrl+Click on blood pools to travel through them, appearing and dissaapearing from the station at will. \
+						Pulling a dead or critical mob while you enter a pool will pull them in with you, allowing you to feast. \
+						You move quickly upon leaving a pool of blood, but the material world will soon sap your strength and leave you sluggish. </B>"
+
+
+/mob/living/simple_animal/slaughter/New()
+	..()
+	spawn()
+		if(src.mind)
+			src.mind.current.verbs += /mob/living/simple_animal/slaughter/proc/bloodPull
+			src.mind.current.verbs += /mob/living/simple_animal/slaughter/proc/slaughterWhisper
+			src << src.playstyle_string
+			src << "<B><span class ='notice'>You are not currently in the same plane of existence as the station. Ctrl+Click a blood pool to manifest.</span></B>"
+			if(!(vialspawned))
+				var/datum/objective/slaughter/objective = new
+				var/datum/objective/demonFluff/fluffObjective = new
+				ticker.mode.traitors |= src.mind
+				objective.owner = src.mind
+				fluffObjective.owner = src.mind
+				//Paradise Port:I added the objective for one spawned like this
+				src.mind.objectives += objective
+				src.mind.objectives += fluffObjective
+				src << "<B>Objective #[1]</B>: [objective.explanation_text]"
+				src << "<B>Objective #[2]</B>: [fluffObjective.explanation_text]"
+
+
+/mob/living/simple_animal/slaughter/Life()
+	..()
+	if(boost<world.time)
+		speed = 1
+	else
+		speed = 0
+
+/mob/living/simple_animal/slaughter/death()
```

Comment:
This proc doesn't exist for us, technically (well it kinda does, because it compiled).

This should be Die()